app/assets/stylesheets/arclight.scss

```diff
@@ -5,3 +5,22 @@ svg {
   height: 1rem;
   width: 1rem;
 }
+body {
```

Comment:
I'm not sure why it's saying the changes from the sticky footer merge are changes, I merged it in last week>